pobbin/api/db/database.py

```diff
@@ -3,7 +3,7 @@
 from sqlalchemy.orm import sessionmaker, Session
 
 
-DOCKER_DB = 'localhost'
+DOCKER_DB = 'pobbin_db'
```

Comment:
we should probably use a config file for this

Reply:
A config file would certainly make it easier to test in different environments. Also it's easier to handle than environment variables. [Configparser](https://docs.python.org/3/library/configparser.html) in the standard library should be sufficent for the current configuration.